### DISCUSSION TITLE
有没有大神开源一个`xmake-examples`，用来展示xmake的各种用法，扩大xmake的影响力？

### DISCUSSION THREAD

**Discussion by shishirong** [xmake-io/xmake#5402] [General]
目前可以学xmake的就只有官方文档和一个收费的视频教程，要是有一个类似[cmake-examples](https://github.com/ttroy50/cmake-examples)的例程让人模仿着开始会好很多。

**Comment by yh-sb** [comment]
There is command `xmake create` with various of options, but I agree that comprehensive nice-looking tutorial would be nice

**Comment by waruqi** [comment]
不是有么，一堆例子了

https://github.com/xmake-examples
https://github.com/xmake-io/xmake/tree/dev/tests/projects

**Comment by shishirong** [comment]
之前搜索xmake-examples，搜索出的结果感觉不够全面。
https://github.com/xmake-io/xmake/tree/dev/tests/projects 这个之前没看到，感觉这个就是我要找的东西。
